homeassistant/components/seventeentrack/__init__.py

```diff
@@ -124,13 +129,73 @@ async def get_packages(call: ServiceCall) -> ServiceResponse:
             ]
         }
 
+    async def add_package(call: ServiceCall) -> ServiceResponse:
+        config_entry_id = call.data[ATTR_CONFIG_ENTRY_ID]
+        tracking_number = call.data.get(ATTR_TRACKING_NUMBER)
```

Comment:
Track number is required according to the schema

```suggestion
        tracking_number = call.data[ATTR_TRACKING_NUMBER]
```